packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -61,7 +70,7 @@ const EmbedPreview = ( {
 	}
 
 	const { provider_url: providerUrl } = preview;
-	const html = 'photo' === type ? getPhotoHtml( preview ) : preview.html;
+	const html = preview.html;
```

Comment:
I realized that for image type embeds the previous code was using the thumbnail instead of the original content which usually is in low resolution.

Reply:
I think `getPhotoHtml` is also completely replacing the `html` returned from the backend. Could there be a reason for that? Should we still use `url` instead of `thumbnail_url` on mobile but keep replacing the `html`?

Reply:
> I think getPhotoHtml is also completely replacing the html returned from the backend.
Yes, it's basically rendering a `img` tag with the image's URL:
```
<p>
  <img src={ imageUrl } alt={ photo.title } width="100%" />
</p>
```

> Could there be a reason for that?
That's a good question, I guess this approach might be related to the fact of providing a controlled HTML with custom styles.

> Should we still use url instead of thumbnail_url on mobile but keep replacing the html?

I recall that originally when I checked a Flickr embed on the web version it was using the full-size URL. However, now I've done another check and I noticed that it's displaying the thumbnail 🤔 . At this stage, I think I'm going to revert this change to match the same behavior as on the web version.